src/libs/comm_py/comm/laserClient.py.ros.in

```diff
@@ -75,8 +75,13 @@ def getLaserClient (jdrc, prefix):
     @return None if Laser is disabled
 
     '''
-    server = jdrc.getConfig().getPropertyWithDefault(prefix+".Server", 0)
+    server = jdrc.getConfig().getProperty(prefix+".Server")
+    if server == "Ice":
+        server=1
+    elif server == "ROS":
+        server=2
+    else server = 0
```

Comment:
you did miss a colon here ```else: server = 0```

Actually you missed that colon in every client, please, review it.